tests/integration.spec.js

```diff
@@ -38,8 +38,14 @@ test(`Foundation homepage`, async ({ page }, testInfo) => {
  *
  * NOTE: this requires a `new-db` run with the seed value set
  *       through RANDOM_SEED=530910203 in your .env file
+ * 
+ * NOTE: This test has the .fixme flag as due to the
+ *       recently added load-in animations for products,
+ *       the PNI search test fails randomly.
+ *       A issue has been filed to fix this at:
+ *       https://github.com/mozilla/foundation.mozilla.org/issues/9373
  */
-test(`PNI search`, async ({ page }, testInfo) => {
+test.fixme(`PNI search`, async ({ page }, testInfo) => {
```

Comment:
Hi @tbrlpld after troubleshooting the CI test failure with @fessehaye, we found that this test is failing inconsistently and we believe that is has to do with the product load in animations. We have filed a issue to get this resolved here: https://github.com/mozilla/foundation.mozilla.org/issues/9373